### ISSUE TITLE
Find a place for "Upgrade Tests" tutorial

### ISSUE THREAD

**Issue by seanmadsen** [civicrm/civicrm-dev-docs#193]
TL;DR: Grab the content from [Upgrade Tests](https://github.com/civicrm/civicrm-buildkit/blob/d20229ac9f63aaf183687157a5e655c953b81f5e/doc/daily-coding.md#upgrade-tests) and find a home for it in the Dev Guide.

Longer story: I'm migrating docs content out of the [buildkit repo](https://github.com/civicrm/civicrm-buildkit) and into the Dev Guide and as part of this process I'm deleting that [file](https://github.com/civicrm/civicrm-buildkit/blob/d20229ac9f63aaf183687157a5e655c953b81f5e/doc/daily-coding.md) which contains a mix of content that needs some work to be well-incorporated into the Dev Guide. I'm using issues to track the process of this migration, section-by-section, since I think the content will likely need to go to multiple places.

**Comment by seanmadsen** [comment]
@totten suggested [here](https://github.com/civicrm/civicrm-dev-docs/pull/223#discussion_r128615202) that perhaps we should put this "upgrade tests" content in `docs/framework/upgrade.md`

**Comment by seamuslee001** [comment]
@seanmadsen i think this can be closed now?

**Comment by seanmadsen** [comment]
ahh yes. Thanks :slightly_smiling_face:
